erpnext/controllers/queries.py

```diff
@@ -224,6 +224,64 @@ def item_query(doctype, txt, searchfield, start, page_len, filters, as_dict=Fals
 			}, as_dict=as_dict)
 
 
+def uom_query(doctype, txt, searchfield, start, page_len, filters, as_dict=False):
+	conditions = []
+	precision = get_field_precision(frappe.get_meta("UOM Conversion Detail").get_field("conversion_factor"))
```

Comment:
Please import this function, check Codacy

Reply:
Added the import 👍